styles/globals.css

```diff
@@ -18,3 +18,10 @@ a {
 * {
   box-sizing: border-box;
 }
+
+@font-face {
+  font-family: "Airbnb Cereal App";
```

Comment:
we might have to change the font we're using later because we aren't sure if we're legally allowed to use the airbnb cereal app but ty for considering details like this!! we'll def use this for setting up font styles

Reply:
sounds good! I'll leave it in there for now then for formatting?

Reply:
yep! we'll talk about restyling things after rachel finalizes assets/fonts on her wireframes so let's leave this be for now